jimmy-pulido-vega/tarea10/src/App.js

```diff
@@ -0,0 +1,47 @@
+import "./App.css";
+import peliculas from "./data/peliculas";
+import Game from "./componets/game/game";
+import { useState } from "react";
+
+function App() {
+  const [nivel, setNivel] = useState(0);
```

Comment:
@jimmyPulido , please use English as main language for programming. Variables,methods, classes, attributes... should be named in english